AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -152,83 +184,147 @@ - (void)_layoutNodes:(NSArray *)nodes atIndexPaths:(NSArray *)indexPaths withAni
   // Block the _editingTransactionQueue from executing a new edit transaction until layout is done & _editingNodes array is updated.
   dispatch_group_wait(layoutGroup, DISPATCH_TIME_FOREVER);
   free(nodeBoundSizes);
-  // Insert finished nodes into data storage
-  [self _insertNodes:nodes atIndexPaths:indexPaths withAnimationOptions:animationOptions];
+
+  if (completionBlock)
```

Comment:
I don't really like the style of mandatory braces for one-line control statements, but it is the style :(